$MirOS: ports/mailnews/pine/patches/patch-pine_other_c,v 1.1 2005/05/04 23:37:41 tg Exp $
--- pine/other.c.orig	Mon Sep 12 22:04:03 2005
+++ pine/other.c	Fri Oct  7 19:12:55 2005
@@ -16922,7 +16922,7 @@ role_config_edit_screen(ps, def, title, 
     if(def && def->patgrp && def->patgrp->category_cmd &&
        def->patgrp->category_cmd[0] && def->patgrp->cat_lim != -1){
 	*apval = (char *) fs_get(20 * sizeof(char));
-	sprintf(*apval, "%d", def->patgrp->cat_lim);
+	sprintf(*apval, "%ld", def->patgrp->cat_lim);
     }
 
     set_current_val(&cat_lim_var, FALSE, FALSE);
